.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "python.linting.flake8Enabled": false,
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true,
+    "python.pythonPath": "/usr/bin/python"
+}
```

Comment:
no new line character at the end of file? also `.vscode`? shouldn't that be ignored? add it to `.gitignore` maybe?

Reply:
Adding